NativeProject/screens/LoginScreen.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import {
+  Button,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+import { WebBrowser, AuthSession } from 'expo';
```

Comment:
@DougLosey Should this be split into separate lines for each part of the import like the import above it?